src/Artifacts.jl

```diff
@@ -14,6 +14,18 @@ function era5_land_forcing_data2021_folder_path(; context = nothing)
     return @clima_artifact("era5_land_forcing_data2021", context)
 end
 
+"""
+    era5_surface_data2008_path(; context)
+
+Return the path to the folder that contains the ERA5 monthly surface data
+
+NOTE: We should be able to use @clima_artifact here, but the folders on central
+seem to be named differently than the artifact name
+"""
+function era5_surface_data2008_path(; context = nothing)
+    return "/groups/esm/ClimaArtifacts/artifacts/era5_surface_fluxes_2008"
```

Comment:
We should be able to use ClimaArtifacts here, but the folder on central has the wrong name and doesn't match the ClimaArtifact. I didn't want to mess with that in case there are other downstreams of this data that shouldn't be interfered with.

Reply:
[arenchon@login3 ~]$ cd /groups/esm/ClimaArtifacts/artifacts/era5_surface_fluxes_2008
[arenchon@login3 era5_surface_fluxes_2008]$ ls
era5_monthly_surface_fluxes_200801-200812.nc

what is the problem?
the filename should be the same as the folder name? (I have no idea but I do get an error)

Reply:
To use ClimaArtifacts the name of the directory should be the artifact name - the artifact name is era5_monthly_averages_2008 so if you copy our CliMA Artifacts output to the toml - 
```
[era5_monthly_averages_2008]
git-tree-sha1 = "c34bbd18a9fd59fd0cd48d1737705255805ab233"
```
 It will look for a directory named era5_monthly_averages_2008 - the folder on the cluster is named era5_surface_fluxes_2008 so it wouldn't find it.

Reply:
https://github.com/CliMA/ClimaLand.jl/blob/kd/ar/calibrate_global_bucket/experiments/calibration/global_bucket/calibrate_bucket_function.jl#L110

@kmdeck it seems we don't have `era5_land_forcing_data2021_folder_path` in ClimaLand.Artifacts anymore? Should I change this to something else?

Reply:
I think #920 removed the 2021 data path function and replaced it with 2008 data